core-bundle/src/Resources/contao/classes/Versions.php

```diff
@@ -651,9 +651,16 @@ public static function addToTemplate(BackendTemplate $objTemplate)
 		$objUser = BackendUser::getInstance();
 		$objDatabase = Database::getInstance();
 
+		$params = array();
+
+		if (!$objUser->isAdmin)
+		{
+			$params[] = $objUser->id;
+		}
+
```

Comment:
Less negation and shorter?
```suggestion
		$params = $objUser->isAdmin ? array() : array($objUser->id);
```

Reply:
Changed in 1f77c2334dddb2a1ca2341f2abb1ac6f49392c48.